### ISSUE TITLE
heketi v10.3.0 fails requiring GLIBC_2.32 -- CentOS has 2.17

### ISSUE THREAD

**Issue by amgads** [heketi/heketi#1869]
### Kind of issue

Bug or Observed behavior:
----------------------
- Installed heketi v10.3.0 in container, but failed to run. with the following error:

      heketi: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by heketi)

- The available glibc from CentOS-7 is "glibc-2.17-324.el7_9.x86_64" and for CentOS-8 "glibc-2.28-127.el8.x86_64"

  Is that a bug or you're getting glibc-2.32 from somewhere else? Does heketi v10.3.0 require EL8 so it doesn't work on EL7?
  Why a go-lang related vulnerability requires to jump to EL8 and all of the sudden drops EL7 support. Besides CentOS EL8 does
  not have glibc 2.32.

  Please confirm CentOS compatibility for 10.3.0 and whether it's for CentOS EL8 only, and where are you getting "glibc-2.32"?

### Information about the environment:

- Heketi version used (e.g. v6.0.0 or master):  v10.3.0
- Operating system used: 
- Heketi compiled from sources, as a package (rpm/deb), or container:  container using the tar package
- If container, which container image: own
- Using kubernetes, openshift, or direct install: k8s
- If kubernetes/openshift, is gluster running inside kubernetes/openshift or outside: gluster running out side a container
- If kubernetes/openshift, how was it deployed (gk-deploy, openshift-ansible, other, custom): ansible

**Comment by muratcabuk** [comment]
same problem with ubuntu 20.04

**Comment by amgads** [comment]
Any clue if this will be addressed soon -- the same issue #1848 is reported on Apr 9th!

**Comment by phlogistonjohn** [comment]
As you've discovered there's already an issue for this, so I hope you understand if we don't want duplicate issues so I'll be closing this one.

As for the timeliness of fixes/releases/etc I'll refer you to our [readme](https://github.com/heketi/heketi/blob/master/README.md) which currently says:
```
As of Heketi release v10.2 the project is considered to be in maintenance only status. This means that only major bugs, performance regressions, or security defects are being considered for inclusion by the project team. Please also expect slow replies to issues.
```

That said, in order to continue to maintain heketi we're looking at switching to go modules in order to support newer version of golang in order to continue to take security fixes in Go. Thus we do expect to do another release this year - as part of the release process we will take the glibc ABI issue into account and make an effort to build heketi such that the binary releases work on a wider variety of base distributions.
